### ISSUE TITLE
Move to Hugo

### ISSUE THREAD

**Issue by kinow** [biouno/biouno.github.io#47]
I've had issues with Jekyll and Ruby on Ubuntu LTS over past years (several times), but never had with Hugo in Apache sites and on my personal site.

@imoutsatsos also had issues with Ruby on Windows, so let's move the site to Hugo, and also add archetypes for the blog post, so that we can document the commands used to create a blog post, and the process to deploy a draft or final article, and how to run everything locally (and faster) with Hugo.

**Comment by kinow** [comment]
Started migrating it today. I selected a theme at random. It rendered the landing page beautifully, but I couldn't migrate the content without breaking URLs, so I gave up after investing some time troubleshooting it - https://github.com/colinwilson/lotusdocs.dev/issues/227

Selected another theme, and now it seems to be working. The theme is not identical to ours, so I will probably have to do some minor adjustments, but for now just trying to migrate everything while keeping the URLs working.
